fiat-web/src/main/java/com/netflix/spinnaker/fiat/config/FiatConfig.java

```diff
@@ -84,6 +96,43 @@ OkClient okClient() {
     val client = okHttpClientConfig.create();
     client.setConnectionPool(new ConnectionPool(maxIdleConnections, keepAliveDurationMs));
     client.setRetryOnConnectionFailure(retryOnConnectionFailure);
+    client.interceptors().add(new RetryingInterceptor(maxElapsedBackoffSeconds));
     return new OkClient(client);
   }
+
+  @Slf4j
+  @AllArgsConstructor
+  private static class RetryingInterceptor implements Interceptor {
+
+    private long maxElapsedBackoffSeconds;
+
+    @Override
+    public Response intercept(Chain chain) throws IOException {
```

Comment:
As of now only GETs, but I added a bit to ensure if someone ever POSTs or PATCHs to not retry.